.github/workflows/commit.yaml

```diff
@@ -91,6 +91,7 @@ jobs:
 
       - name: "Upload coverage report"  # only on master push and only once (not per OS)
         if: github.event_name == 'push' && github.ref == 'refs/heads/master' && runner.os == 'Linux'
-        env:
-          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
-        run: bash <(curl -s https://codecov.io/bash)
+        uses: codecov/codecov-action@v2
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
BTW for OSS projects token shouldn't be needed when using the action I think. One less secret to manage

Reply:
will verify here then propagate. https://github.com/tetratelabs/car/pull/19

Reply:
verified in car and https://github.com/tetratelabs/func-e/pull/431 will do it here